flake.nix

```diff
@@ -0,0 +1,31 @@
+{
+  description = "Hardware and Firmware of the Solo Quadruped Master Board.";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+  };
+
+  outputs =
+    { self, nixpkgs }:
+    let
+      pkgs = nixpkgs.legacyPackages.x86_64-linux;
+    in
+    {
+      packages.x86_64-linux.sdk = pkgs.stdenv.mkDerivation {
```

Comment:
if you name the package `default`, that will trigger some nice shortcuts, like `nix build` instead of `nix build .#sdk`

Reply:
yes, but i don't really know what the "default" is for that repo, is the sdk more "important" than the rest?

Reply:
Yes, I think it is.
The firmware would most of the time just be downloaded as a binary, eg. from github release or ci